test_scripts/ATF_ShutDown_IGNITION_OFF.lua

```diff
@@ -0,0 +1,70 @@
+--  Requirement summary:
+--  Check that SDL finish its work properly by ignition off with one registered Application
+--
+--  Description:
+--  1. HMI is running
+--  2. One App is registered and activated on HMI
+--  2. Perform ignition Off
+--
+--  Expected behavior:
+--  1. HMI sends OnExitAllApplications(IGNITION_OFF)
+--  2. SDL sends to HMI OnSDLClose 
+--  3. SDL sends to App OnAppInterfaceUnregistered and stops working
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+function Test:Start_SDL_With_One_Activated_App()
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self):Do(function()
+    self:initHMI():Do(function()
+      commonFunctions:userPrint(35, "HMI initialized")
+      self:initHMI_onReady():Do(function ()
+        commonFunctions:userPrint(35, "HMI is ready")
+        self:connectMobile():Do(function ()
+          commonFunctions:userPrint(35, "Mobile Connected")
+          self:startSession():Do(function ()
+            commonSteps:ActivateAppInSpecificLevel(self, 
+              self.applications[config.application1.registerAppInterfaceParams.appName])
+            EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+          end)
+        end)
+      end)
+    end)
+  end)
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Check that SDL finish it's work properly by IGNITION_OFF")
+function Test:ShutDown_IGNITION_OFF()
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "SUSPEND" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLPersistenceComplete")
+
+  self.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "IGNITION_OFF" })
+  EXPECT_NOTIFICATION("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = false }) 
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose"):Do(function ()
```

Comment:
@istoimenova 
Times(0) won't show anything in this case - the function will be finished before OnSDLClose() be received. Adding DelayedExp() will be not correct because it will increase time of test running without special necessity. So current implementation is correct